### PR TITLE
docs(readme.md): refactor readme to be more focused, added logo

### PR DESCRIPTION
**Pull request by rubinovitz**
This is an opinionated edit so I'm curious about your feedback

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,5 +1,7 @@
 # Homebase React
 
+![Homebase React Logo](public/images/logo-blk.png)
```
Comment (smothers) [inline README.md:3]: I don't see this image in the preview

File: README.md [hunk 2]
```diff
@@ -8,39 +10,25 @@
 
 *The graph database for delightful React state management*
 
-## Installation
```
Comment (smothers) [inline README.md:11]: I think this should stay at the top. I often want a reminder of the install instructions even for packages I'm already using in other projects. And I get annoyed by readmes that burry these and make me read

File: README.md [hunk 3]
```diff
@@ -8,39 +10,25 @@
...
-Homebase React lets you plug a relational database into your React application with just 3 lines of code. In fact, it's the same database that powers Roam Research and many other ClojureScript applications, but with an API that's familiar to React and JS developers.
 
-- Replace Redux with something simpler and more declarative
-- Stop spending time wiring up custom datatypes, reducers, caches and other bespoke state mumbo jumbo
-- Gain all the benefits of relational data like a single source of truth
-- Traverse your data graph like it's one big JSON object
-- Query your database with a convenient JSON query syntax
-- Query your database with Clojure style [Datalog](https://docs.datomic.com/on-prem/query.html) if you need more power
-- It's just data
+Homebase React makes state management painless by enabling you to plug a relational graph database into your React application with just 3 lines of code. This is the same database that powers Roam Research and many other ClojureScript applications, but with an API that's familiar to React and JS developers.
```
Comment (smothers) [inline README.md:14]: A bit more Hemingway'd

Homebase React makes state management painless. Plug a graph database into your React application with 3 lines of code. This is the same database that powers Roam Research and many other ClojureScript applications, but with an API that works for React and JS developers.

File: README.md [hunk 4]
```diff
@@ -8,39 +10,25 @@
...
 
-The goal of Homebase React is to be immediately more intuitive than any denormalized JSON store and over time we will eliminate all the tedious aspects of manipulating data on clients. Homebase React makes it easier to work with complex data on client by making our applications [local-first](https://news.ycombinator.com/item?id=21581444).
+## Features
```
Comment (smothers) [inline README.md:16]: 👍🏽

File: README.md [hunk 5]
```diff
@@ -8,39 +10,25 @@
...
+- The simplest and most declarative state management solution
+- The power of a backend relational graph database, but without having to wait on the network
+- Convenient JSON query syntax
+- Powerful Clojure style [Datalog](https://docs.datomic.com/on-prem/query.html) query syntax if you need it
+- Traverse your data graph like it's a big JSON object
+- Backup your data to the cloud
 
-## Examples
```
Comment (smothers) [inline README.md:37]: I want this up higher. Maybe right below the installation instructions. This is a chance to show off, and I don't think we should prioritize telling if we can show

File: README.md [hunk 6]
```diff
@@ -8,39 +10,25 @@
...
-### Live Demos
-You can see our hosted live demos [here](https://homebaseio.github.io/homebase-react)
-
-### Code Examples
-You can clone and run our React code examples [here](https://github.com/homebaseio/homebase-react-examples).
+## Roadmap
 
+1. Document integration with backends
```
Comment (smothers) [inline README.md:26]: Since we just added Firebase change to

Document integration with more backends

File: README.md [hunk 7]
```diff
@@ -8,39 +10,25 @@
...
+- Traverse your data graph like it's a big JSON object
+- Backup your data to the cloud
 
-## Examples
-### Live Demos
-You can see our hosted live demos [here](https://homebaseio.github.io/homebase-react)
-
-### Code Examples
-You can clone and run our React code examples [here](https://github.com/homebaseio/homebase-react-examples).
+## Roadmap
```
Comment (smothers) [inline README.md:24]: 👍🏽

File: README.md [hunk 8]
```diff
@@ -52,13 +40,12 @@ The HomebaseProvider wraps your React app and makes a relational database access
 import { HomebaseProvider, useEntity, useTransact, useQuery } from 'homebase-react'
 
 const config = {
-  // Schema is not a type system like in most DBs. 
-  // That is something we're considering, but for now it is 
-  // mainly applied at query time to simplify relational queries.
-  // The only schema currently supported is:
+  // Schema is not a type system,
```
Comment (smothers) [inline README.md:43]: 👍🏽

File: README.md [hunk 9]
```diff
@@ -132,9 +119,9 @@ transact([['retractEntity', 1]])
 
 ### `useQuery`
 
-Use queries to return an array of entities that meet a given criteria. Our query API is powered by datalog, but exposed as JSON similar to a JS SQL driver or Mongo DB. Datalog is similar to SQL and is incredibly powerful. However only a subset of features are currently available in JSON. 
+Use queries to return an array of entities that meet a given criteria. Our query API is powered by Datalog, but exposed as JSON similar to a JS SQL driver or MongoDB. Datalog is similar to SQL and is incredibly powerful. However, only a subset of features are currently available in JSON.
 
-We are very interested in what features the community wants, and will prioritize based on feedback. In the meantime you can further filter results with JS `filter()` and `sort()`.
+We will prioritize features based on community feedback so please open an issue if there's something you need. In the meantime you can further filter results with JS `filter()` and `sort()`.
```
Comment (smothers) [inline README.md:124]: 👍🏽

**Comment by smothers** [comment]
@rubinovitz love where you're taking this. I've got a bunch of opinions so let me know what you think. Generally, I want to prioritize letting people get in and out without reading. Otherwise, I like all the changes.

File: README.md (hunk 1 above)
Comment (rubinovitz) [reply README.md:3]: yikes thanks for pointing that out. committed it and pushed now

File: README.md (hunk 2 above)
Comment (rubinovitz) [reply README.md:11]: makes sense! I tried moving it under 'features' right now based on the format I've been seeing in popular projects I like of pitching the library before giving the install instructions. Is that high enough?

File: README.md (hunk 3 above)
Comment (rubinovitz) [reply README.md:14]: I don't think it's obvious that the first two sentences are connected.

Comment (rubinovitz) [reply README.md:14]: I don't think it's obvious to people that using a database makes state management easier. So I try to posit that idea here and then prove it with the features section.

File: README.md (hunk 2 above)
Comment (rubinovitz) [reply README.md:11]: ok I ended up moving it under roadmap as well but I'm still split on it. I do think we need to pitch what we're doing more better and faster and I think we should prioritize that given the goal right now is to get new users. Does that make sense or still bothered?

File: README.md (hunk 3 above)
Comment (rubinovitz) [reply README.md:14]: I can also try to make the sentence with connected ideas shorter if that's what you don't like. I'm personally not a Hemmingway fan, but I am a Stephen King fan.

**Review by smothers — APPROVED** [review]

**Pull request merged**